invenio_theme_tugraz/assets/semantic-ui/less/invenio_theme_tugraz/header.less

```diff
@@ -190,7 +190,7 @@ svg:not(:root) {
 }
 
 .main-menu-underline-line {
-  border: 2px solid #e4154b !important;
```

Comment:
Please add/change these colors to this [Issue](https://github.com/mb-wali/invenio-theme-tugraz/issues/76).